README.md

```diff
@@ -20,10 +20,8 @@ Our app will have two pages:
 
 You'll notice that there are a number of `SessionsController` specs failing because, principally, no `SessionsController` exists.
 
-`rails generate controller Sessions new` to create a SessionsController with a `new` method. Controller method scaffolding is not generally that useful (you too can type `def new`!), but including `new` makes the generator create views, which we will also want.
+Run `rails generate controller Sessions new` to create a SessionsController with a `new` method. Controller method scaffolding is not generally that useful (you too can type `def new`!), but including `new` makes the generator create views, which we will also want.
```

Comment:
a few things about this: 

running this generates a spec file for the sessions controller. the specs for this lab are in session_controller_spec. this is a bit confusing to have two spec files. i would have the lab test file be sessions_controller_spec, to correspond with the controller they will be building. we could either have them run with the flag to skip specs, or have them be prompted by the task to skip generating the spec file. not sure which would be better. leaning towards the first option, so they don't accidentally overwrite the lab's spec file. thoughts @jmburges ?

second, the view sessions/new.html.erb already existed when i ran this, so we probably want to delete it on the master branch if we have them running this generator task to show how this file is generated, as stated in the readme.